k8s/installation/observability.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Monitoring the Kubernetes Operator
+sidebarTitle: Observability
+description: Learn how to enable and customize ngrok's Kubernetes ingress Observability features.
+---
+
+Configuring metrics and logging for the ngrok Kubernetes Operator is essential for gaining visibility into how the operator is handling your ingress resources and other CRDs.
+
+This guide explains the available observability features, and how to enable and customize them to:
+- Diagnose issues more quickly.
+- Track system performance.
+- Ensure the operator is running reliably in production.
+
+## Metrics
+
+The ngrok Kubernetes Operator exposes a set of [Prometheus-compatible](https://prometheus.io/) metrics that provide insight into the operator’s internal state and the resources it manages. By enabling metrics, you can monitor reconciliation activity, track resource health, and build alerts around key operational behaviors. 
+
+### Supported metrics
+
+This project is built using [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder?tab=readme-ov-file#kubebuilder). By default, it exposes the metrics described in the Kubebuilder [metrics reference](https://book.kubebuilder.io/reference/metrics-reference.html?highlight=metrics#default-exported-metrics-references).
+
+### Accessing metrics
+This operator exposes prometheus metrics on the `/metrics` endpoint. The metrics are exposed on the `:8080` port and can be scraped by Prometheus or other services.
+
+## Logging
+
+The ngrok Kubernetes Operator generates structured logs that capture its reconciliation processes, configuration changes, and any errors encountered while managing resources. These logs are a primary source of truth when diagnosing issues or understanding the operator’s actions.
+
+### Configure log levels
+
+By tuning the operator’s log level, you can control the level of detail emitted in logs. This lets you strike the right balance between observability and signal-to-noise, whether you’re troubleshooting an issue, monitoring production workloads, or reducing log volume.
+
+### Available log levels
```

Comment:
--zap-log-level level                    Zap Level to configure the verbosity of logging. Can be one of 'debug', 'info', 'error', or any integer value > 0 which corresponds to custom debug levels of increasing verbosity